app.vue

```diff
@@ -44,10 +44,8 @@ app.vueApp.use(datagouv, {
 
 useHeadSafe({
   htmlAttrs: {
-    lang: i18nHead.value.htmlAttrs!.lang,
+    lang: locale,
   },
-  link: [...(i18nHead.value.link || [])],
-  meta: [...(i18nHead.value.meta || [])],
```

Comment:
This one was adding alternative pages for SEO purpose, for multilingual website this is required but it has no usage for us with only one language (more on the subject on https://developers.google.com/search/docs/specialty/international/localized-versions?hl=fr)